app/src/main/res/color/button_selector_color.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_selected="true" android:state_focused="true" android:color="@color/black"/>
```

Comment:
Have you tested this on white mode?

Reply:
I have not, I will do

Reply:
It works, although white mode is generally unusable on TV.